client/index.html

```diff
@@ -5,9 +5,98 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="css/styles.css">
-    <title>co-habit</title>
+    <link rel="stylesheet" href="css/navStyles.css">
+    <!-- Import Allerta fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Allerta&display=swap" rel="stylesheet">
+    <!-- Add icon library -->
+    <script src="https://kit.fontawesome.com/490bacaf0b.js" crossorigin="anonymous"></script>
+    <link rel="icon" href="./images/grabbit-icon.jpg" />
+    <title>Cohabit</title>
 </head>
 <body>
-    
+    <header id="top">
+        <section class="top-bar">
+            <button class="dark-mode-btn">Switch to dark mode</button>
+            <nav class="nav-bar">            
+                <a class="active" id="register"><i class="fas fa-user-plus"></i></a>
+                <a id="login"><i class="fas fa-sign-in-alt"></i></a>
+            </nav>
+        </section>
+
+
+        <div class="heading">
+            <div id="title-icon">
+                <i class="fas fa-fist-raised" id="title-icon"></i>
+            </div>
+            <div id="title">
+                <h1 id="app-name">grabbit</h1>
+                <h5 id="tagline">habbit your way</h3>
+            </div>            
+        </div>
+    </header>
+
+    <main>
+        <div class="auth-btns">
+            <button class="login-btn">login</button>
+            <button class="register-btn">register</button>            
+        </div>
+
+        <form class="auth-form">
```

Comment:
if we're doing single page application might better to keep page blank and render all the forms with js? 

its possible to keep the login at the front, but this means we have to hide the login form every time other elements are needed e.g main feed, register, etc etc

Reply:
Good point.  I initially thought it wouldn't be too much hassle to just do a body.innerHTML=""  (or something similar) at the start of each rendering function but I'm happy refactor all of this once I've made sure everything looks the way it's supposed to.  :)